src/test/java/org/inferred/freebuilder/processor/MapSourceTest.java

```diff
@@ -62,29 +62,25 @@ public void test_guava_j6() {
         "  /**",
         "   * Associates {@code key} with {@code value} in the map to be returned from",
         "   * {@link Person#getName()}.",
-        "   * Duplicate keys are not allowed.",
+        "   * If the map previously contained a mapping for the key,",
+        "   * the old value is replaced by the specified value.",
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String literal "   * the old value is replaced by the specified value." appears 4 times in this file; the first occurrence is on line 66](https://www.codacy.com/app/Chris-Purcell-39/FreeBuilder/file/2409332806/issues/source?bid=3146243&fileBranchId=3234738#l66)